### PR TITLE
Refactoring changes in Issued_book page due to restructuring of firebase

### PR DESCRIPTION
**Pull request by spmshakti**
### Refactoring changes due to restructuring of firebase
pbook

1. Data is fetched from firebase: user=>user=>Account
2. If is returned is true it shows in previous book ,else it shows in Issued section.

### CODE REVIEW

**Comment by spmshakti** [comment]
@piyuuussh @ShubhamTiwari55

File: lib/books model/privious_issuedbook_model.dart
Comment (piyuuussh) [inline lib/books model/privious_issuedbook_model.dart:1]: Why is this model of previous book is created, we have to call the account collection and book id as doc in the books. Dont create another model

File: lib/api.dart
Comment (piyuuussh) [inline lib/api.dart:1]: In the user collection in the doc call firebasefirestore _auth.currentusercredential.uid for calling the current user account details

**Review by piyuuussh — CHANGES_REQUESTED** [review]
Make all the necessary changes.

File: lib/books model/privious_issuedbook_model.dart
Comment (spmshakti) [reply lib/books model/privious_issuedbook_model.dart:1]: previous book model cleared

File: lib/api.dart
Comment (spmshakti) [reply lib/api.dart:1]: Necessary changes done

**Comment by spmshakti** [comment]
@piyuuussh  review the changes

**Comment by piyuuussh** [comment]
The issue book page is still not functional... Make the necessary changes
image

**Review by piyuuussh — APPROVED** [review]
looks good

**Pull request merged**